package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "soakinthatlane.business",
+  "version": "0.0.1",
+  "description": "ayyy",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/internet-research-labs/soakinthatlane.business"
+  },
+  "keywords": [
+    "soak",
+    "dat"
+  ],
+  "author": "IRL",
+  "license": "IRL",
+  "bugs": {
+    "url": "https://github.com/internet-research-labs/soakinthatlane.business/issues"
+  },
+  "homepage": "https://github.com/internet-research-labs/soakinthatlane.business",
+  "devDependencies": {
+    "autoprefixer": "^6.3.7",
+    "gulp": "^3.9.1",
+    "gulp-browserify": "^0.5.1",
+    "gulp-concat": "^2.6.0",
+    "gulp-plumber": "^1.1.0",
+    "gulp-postcss": "^6.1.1",
+    "gulp-rename": "^1.2.2",
+    "gulp-sass": "^2.3.2"
+  }
+}
```

Comment:
Going forward, we're moving to webpack, so we should be looking to switch some of this out.

Should we go strictly for npm scripts?